wire-ios-data-model/Source/ManagedObjectContext/NSManagedObjectContext+BackupImport.swift

```diff
@@ -21,7 +21,7 @@ import Foundation
 extension NSManagedObjectContext {
 
     /// Prepare a backed up database for being imported, deleting self client, push token etc.
-    func prepareToImportBackup() {
+    func prepareToImportBackup() { // TODO: is this relevant for the import use case?
```

Comment:
yes! good catch we didn't think about the other values: pushToken, pushKitToken.

For the lastEventId, I think we can keep it to nil, and then perform a slowsync

Reply:
Actually I noticed that pushToken and pushKitToken are not used anymore. I created a PR: https://github.com/wireapp/wire-ios/pull/2403